mgmtsystem_security_event/views/security_event_measure.xml

```diff
@@ -0,0 +1,57 @@
+<openerp>
```

Comment:
Please use the full model name for this file `mgmtsystem_security_event_measure.xml`